lib/github_changelog_generator/generator/generator_fetcher.rb

```diff
@@ -96,6 +99,16 @@ def associate_tagged_prs(tags, prs, total)
       end
     end
 
+    def try_merge_commit_sha_from_gitlab(merge_request)
+      merge_commit_sha = nil
+      if merge_request.key?("merge_commit_sha")
+        merge_commit_sha = merge_request["merge_commit_sha"]
+      elsif merge_request["events"] && (event = merge_request["events"].find { |e| e["event"] == "merged" })
+        merge_commit_sha = event["commit_id"]
+      end
+      merge_commit_sha
+    end
```

Comment:
`if` is an expression: this can be the whole body.

```ruby
      if merge_request.key?("merge_commit_sha")
        merge_request["merge_commit_sha"]
      elsif merge_request["events"] && (event = merge_request["events"].find { |e| e["event"] == "merged" })
        event["commit_id"]
      end
```